index.html

```diff
@@ -0,0 +1,158 @@
+        </div>
+        <div class="right-block">
+            <div class="my-photo">
+            </div>
+            
+            <div class="latest-articles">
+                <div class="mini-block"><div class="mini-block-photo"><img src="./assets/a.jpg" class="krez" height="370px" width="340px"></div><h1>Trying out Vue version 3.0 </h1>
+                    <h2>When it comes to client web projects my favourite 
+                        frontend frameworks are by far Angular and Vue. So some weeks 
+                        ago I stworking on a fresh client project with 
+                        Vue v2 and Typescript. However..</h2>
+                    </div>
+            </div>
+        </div>
+        <div class="my-info" id="about-me">
+            <h1>About me</h1>
+            <h2>It was back in 2016, when I first got my hands dirty with software.
+                 Since then my future was defined. <br>
+                 In university my speciality is "Microprocessor information management systems"
+                  and during my studies I learned<br>
+                  oop principlesnt, JS, C, C++, Java, Git, HTML/CSS, PostgerSQL, I fell  in love with the beauty of web.</h2>
+                    <h3>I visited "IT-class" and Java Epam courses and i will finish my university in 2021 and want to work like Junior Software Engineer</h3>
+            <div class="button1">Check out my portfolio</div>
+        </div>
+        <div class="career" id = "career">
+            <div class="career-ages">
+                <h1>Career</h1>
+                <div class="circle">
+                    <div class="minicirlce"></div>
+                </div>
+                <div class="circle">
+                    <div class="minicirlce"></div>
+                </div>
+                <div class="circle">
+                    <div class="minicirlce"></div>
+                </div>
+                <div class="circle">
+                    <div class="minicirlce"></div>
+                </div>
+            </div>
+            <div class="carrer-info">
+                <div class="career-time" style="margin-top: 120px;"><strong>2016 - 2018 </strong> &nbsp;During this period I studed in university and learned programming basic.</div>
+                <div class="career-time"><strong>2019 </strong> &nbsp;During this period I visited "IT-class" courses and learned C, C++, HTML/CSSm PHP.</div>
+                <div class="career-time"><strong>2019 - 2020 </strong> &nbsp;During this period I visited Epam Java training and learned Java, made tasks in EPAM lanoratory; I made project on PostgreSQL</div>
+                <div class="career-time"><strong>2020 </strong> &nbsp;Now i pass internship in IBA Homel and learn Git, JS, Angular</div>
+            </div>
+        </div>
+        <div class="skills">
+            <h1>My skills</h1>
+            <div class="skill-block" style="margin-left: 150px;"><img src="./assets/js.jpg" ><h2>JavaScript</h2></div>
```

Comment:
please replace all jpg images to png. Better to have icons with invisible background